src/db.c

```diff
@@ -1974,6 +2027,25 @@ unsigned int getKeysInSlot(unsigned int hashslot, robj **keys, unsigned int coun
     return j;
 }
 
+/* Populate the specified array of objects with keys in the specified slot.
+ * New objects are returned to represent keys, it's up to the caller to
+ * decrement the reference count to release the keys names. */
+void getChannelsInSlot(unsigned int hashslot, robj **keys) {
```

Comment:
Unlike getKeysInSlot, it doesn't like like we ever strictly return string objects (which is why this function returns string objects). We could just call rax recursiveFree with the callback that disconnects all the clients.

Reply:
That's true. I was planning to add a command like `CLUSTER GETCHANNELSINSLOT` later on. Do you think it will be meaningful information for the user? Like before slot migration verifying if there are any slots with active channels.

Reply:
I don't think so? There is no way to list all of the channels today right? I don't see a compelling reason to ever need to do that.

Reply:
As discussed, making it inplace to avoid exposing it unless required.